server/BraveAlerterConfigurator.js

```diff
@@ -261,6 +299,10 @@ class BraveAlerterConfigurator {
     return returnMessage
   }
 
+  getClientMessageForRequestToReset(language) {
```

Comment:
You could add a new file to test this new function called `test/unit/BraveAlerterConfigurator/getClientMessageForRequestToResetTest.js`.